docs/whatsnew/1.4.0/1_4_0.rst

```diff
@@ -18,4 +18,34 @@ Check out the :ref:`hmm-section` tutorial and the
 
 .. figure:: hmm_hairpin.png
 
-   HMM analysis of a tethered DNA hairpin held at three different bead separations.
\ No newline at end of file
+   HMM analysis of a tethered DNA hairpin held at three different bead separations.
+
+Automatic bead cropping
+-----------------------
+
+Added :meth:`~lumicks.pylake.kymo.Kymo.estimate_bead_edges()` and :meth:`~lumicks.pylake.kymo.Kymo.crop_beads()` for roughly cropping the beads out of a kymograph. This can help when batch processing of kymographs.
+
+.. figure:: bead_edges.png
+
+Filter customization kymotracking
+---------------------------------
+
+We added the option to customize the filters applied prior to peak detection to :func:`~lumicks.pylake.track_greedy`.
+To do this, we added two additional parameters:
+
+- `filter_width` allows customizing the filter applied prior to detection.
+- `adjacency_filter` applies a filter on the detected peaks, removing any fluorescent peaks that do not have a detected peak in an adjacent frame.
+
+This allows using lower thresholds, while keeping false detections in check.
+
+.. figure:: tracking_comparison.png
+
+.. figure:: tracking_comparison_threshold.png
```

Comment:
could you remake the plots using something like `viridis` for the colormap? I don't know if it's the colorblindness but I can barely distinguish the tracks on the green background. The perceptually uniform colormaps are anyways best practice for data visualization (at least in a single-channel format)

Reply:
Good point. Switched to lime on gray.